sdk/containerregistry/container-registry/src/generated/models/parameters.ts

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is regenerated.
+ */
+
+import {
+  OperationParameter,
+  OperationURLParameter,
+  OperationQueryParameter
+} from "@azure/core-http";
+import {
+  Manifest as ManifestMapper,
+  ChangeableAttributes as ChangeableAttributesMapper,
+  Paths108HwamOauth2ExchangePostRequestbodyContentApplicationXWwwFormUrlencodedSchema as Paths108HwamOauth2ExchangePostRequestbodyContentApplicationXWwwFormUrlencodedSchemaMapper,
+  PathsV3R3RxOauth2TokenPostRequestbodyContentApplicationXWwwFormUrlencodedSchema as PathsV3R3RxOauth2TokenPostRequestbodyContentApplicationXWwwFormUrlencodedSchemaMapper
```

Comment:
Whoa, I know these are generated but those are some gnarly names!

Reply:
I felt the same too!

Reply:
Very impressive. Borderline even by Java standards. 😂